back/engines/commercial/project_management/spec/policies/project_policy_spec.rb

```diff
@@ -41,6 +42,7 @@
       it { is_expected.to permit(:update) }
       it { is_expected.to permit(:reorder) }
       it { is_expected.not_to permit(:destroy) }
+      it { is_expected.to permit(:delete_inputs) }
```

Comment:
Shouldn't we add expectations everywhere in this file? Otherwise we do not have a description of what the behaviour is in other situations.

Reply:
Yes, but the problem is that in the other cases, it is not clear whether a timeline or continuous project is used and therefore the behavior is undefined in those cases. I could add the other cases but then it would take some effort to make it clear what type of project is used. I would have to think about this, because I'm not happy with the current solution. I think ideally, I would like to combine two factories. If this is not possible, it feels to me that the whole projects factory needs refactoring.

Reply:
I would not go that far :-) Then I propose not to do what I suggested.